windows/progress-view/ProgressViewView.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "JSValueXaml.h"
+#include "ProgressViewView.h"
+#include "ProgressViewView.g.cpp"
+
+namespace winrt {
+    using namespace Microsoft::ReactNative;
+    using namespace Windows::Foundation;
+    using namespace Windows::Foundation::Metadata;
+    using namespace Windows::UI;
+    using namespace Windows::UI::Xaml;
+    using namespace Windows::UI::Xaml::Controls;
+    using namespace Windows::UI::Xaml::Input;
+    using namespace Windows::UI::Xaml::Media;
+    using namespace Windows::UI::Xaml::Media::Imaging;
+} // namespace winrt
+
+namespace winrt::progress_view::implementation {
+
+    ProgressViewView::ProgressViewView(winrt::IReactContext const& reactContext) : m_reactContext(reactContext) {
+        this->Minimum(0);
+        this->Maximum(100);
+    }
+
+    void ProgressViewView::UpdateProperties(winrt::IJSValueReader const& reader) {
+        auto const& propertyMap = JSValueObject::ReadFrom(reader);
+
+        for (auto const& pair : propertyMap) {
+            auto const& propertyName = pair.first;
+            auto const& propertyValue = pair.second;
+
+            if (propertyName == "progress") {
+                this->Value(propertyValue.AsDouble() * 100);
+            }
+            else if (propertyName == "progressTintColor") {
+                this->Foreground(propertyValue.To<winrt::Brush>());
+
+            }
+            else if (propertyName == "trackTintColor") {
+                this->Background(propertyValue.To<winrt::Brush>());
+            }
+            else if (propertyName == "progressViewStyle") {
+                if (propertyValue == "bar") {
+                    this->Background(SolidColorBrush(ColorHelper::FromArgb(000, 000, 000, 000)));
```

Comment:
Can there be other color values when using this prop?

Reply:
There wasn't much info on this property (The description being "The progress bar style.") but by the picture I assumed it was making the background transparent. There could be other color values but the property 'trackTintColor' handles that.

Reply:
Yeah, I get that. The maintainers might be able to clarify that, too.